kjourney/src/app/filter/filter.component.html

```diff
@@ -0,0 +1 @@
+<p>filter works!</p>
```

Comment:
have a todo here?

Reply:
filter was added but not enough time to be implemented